extensions_admin/pulp_docker/extensions/admin/manifest.py

```diff
@@ -0,0 +1,118 @@
+from gettext import gettext as _
+
+from pulp.client.commands import options
+from pulp.client.commands.criteria import DisplayUnitAssociationsCommand
+from pulp.client.commands.unit import UnitCopyCommand, UnitRemoveCommand
+
+from pulp_docker.common.models import Manifest, Blob
+
+DESC_COPY = _('copies manifests from one repository into another')
+DESC_REMOVE = _('remove manifests from a repository')
+DESC_SEARCH = _('search for manifests in a repository')
+FORMAT_ERR = _('The docker manifest formatter can not process %s units.')
+
+UNIT_ID_TEMPLATE = '%(digest)s'
```

Comment:
Consider using the new Python 3 compatible format syntax. For this statement, it would be something like this:

```
UNIT_ID_TEMPLATE = '{digest}'
```

Then later when you try to use it, you can format it like this:

```
print UNIT_ID_TEMPLATE.format(digest='my_digest')
```

Doing this now will make it easier to transition to Python 3 later. It's definitely fine if you don't want to make the change though, I just wanted to put the thought out there at least maybe for the other new code we write.

https://docs.python.org/2/library/string.html#formatstrings